source/Ocl/Ocl.csproj

```diff
@@ -12,4 +12,8 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Sprache" Version="2.3.0" />
```

Comment:
Do we need net40 support? If we don't there are a few other libraries that can offer better performance or better error reporting.

Reply:
`netcoreapp` should be enough